### DISCUSSION TITLE
How many times has Krusty the Clown from The Simpsons been married?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#246] [Galaxy Trivia] [answered]
**Hint:** 🕐🕔

**Comment by thuongtruong1009** [comment]
Krusty the Clown from The Simpsons has been married twice. His first marriage was to a woman named "Princess Penelope," but the marriage was short-lived and ended in divorce. His second marriage was to a woman named "Gloria," who was later revealed to be a convicted felon. This marriage also ended in divorce.

**Reply by ParanoidUser** [reply]
Nice try! But don't underestimate Krusty, he's been married more than twice. Can you guess?

**Comment by live627** [comment, marked as answer]
I see two clocks, one at 000 and the other at 015. So... fifteen.
